src/App.jsx

```diff
@@ -160,18 +152,14 @@ function App() {
     // don't load ANY details until wallet is Checked
     if (walletChecked) {
       loadDetails("app");
-      loadDetails("account");
-      loadDetails("userBonds");
     }
   }, [walletChecked]);
 
   // this useEffect picks up any time a user Connects via the button
   useEffect(() => {
     // don't load ANY details until wallet is Connected
     if (connected) {
-      loadDetails("app");
       loadDetails("account");
-      loadDetails("userBonds");
```

Comment:
these calls were duplicating & unnecessary. After the above changes the app loads successfully for both connected & unconnected users